pkg/actuators/machine/reconciler.go

```diff
@@ -211,12 +211,19 @@ func (r *Reconciler) update() error {
 	runningLen := len(runningInstances)
 	var newestInstance *ec2.Instance
 
+	//Prepare the tag list with infrastructure tags.
+	//these tags will update to the ec2 instance.
+	tagList, err := r.prepareTags()
+	if err != nil {
+		return err
+	}
+
 	if runningLen > 0 {
 		// It would be very unusual to have more than one here, but it is
 		// possible if someone manually provisions a machine with same tag name.
 		klog.Infof("%s: found %d running instances for machine", r.machine.Name, runningLen)
 		newestInstance = runningInstances[0]
-
+		//newestInstance.Tags = tagList
```

Comment:
Guessing this isn't needed?

Reply:
this is required, we have to un-comment,

Reply:
yes. taken